spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/h2/V2__TASK_MANIFEST_SETUP.sql

```diff
@@ -0,0 +1,10 @@
+create table task_execution_metadata (
```

Comment:
For embedded database we use only one schema version(to make startup faster) and anyway db is always recreated anyway, so just add stuff to `V1__INITIAL_SETUP.sql`.

Reply:
Done